docs/network/networkdevice/bluesocket.asciidoc

```diff
@@ -0,0 +1,64 @@
+// to display images directly on GitHub
+ifdef::env-github[]
+:encoding: UTF-8
+:lang: en
+:doctype: book
+:toc: left
+:imagesdir: ../../images
+endif::[]
+
+////
+
+    This file is part of the PacketFence project.
+
+    See PacketFence_Network_Devices_Configuration_Guide.asciidoc
+    for authors, copyright and license information.
+
+////
+
+
+//=== Bluesocket
+
+==== MAC Authentication
+
+===== Bluesocket side
+
+In order to configure mac authentication on the Bluesocket, you must have access to the controller.
+
+First, you must configure a RadiusWebAuthServer in External Authentication and enable "Enable Radius MAC Authentication" and add Authentications rules.
+
+This Authentication Rules needs to match the PacketFence role you define to the Bluesocket role you defined.
+
+image::bluesocket-radiuswebserver.png[scaledwidth="100%",alt="Bluesocket radius web server"]
+
+Next you have to create an SSID in Wireless -> SSIDs and important, check for "Enable Captive Portal Authentication".
```

Comment:
Next, you need to create an SSID in `Wireless -> SSIDs` and important, check for "Enable Captive Portal Authentication".